dragonfly/implementations/psyllid_provider.py

```diff
@@ -218,18 +68,18 @@ def request_status(self, channel):
         logger.info('Checking Psyllid status of channel {}'.format(channel))
         try:
             result = self.provider.get(self.queue_dict[channel]+'.daq-status', timeout=10)
-            logger.info(result)
+        except:
+            logger.warning('Psyllid instance for channel {} is not running or sth. else is wrong'.format(channel))
+            self.status_dict[channel]=None
+            self.status_value_dict[channel]=None
+            return self.status_value_dict[channel]
+        else:
```

Comment:
The except should be qualified, after that you should consider if you want a `finally` statement to do some cleanup (see  https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions) before that exception is raised